### DISCUSSION TITLE
Prepared search questions

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#5016] [Ideas]
GloBI has their search page set up as a question.

![image](https://user-images.githubusercontent.com/5725767/188733103-43c2d4fe-4923-4cdc-9e9e-921a420ed731.png)

Maybe we could set up a search page like this with several interesting questions?

**Comment by ewommack** [comment]
I like it. We would want to make sure we represented all of the different types of collections present. Would a question on how to find the bus at UAM by ok @AJLinn? I vote for that one if it is.

**Reply by AJLinn** [reply]
I'd be okay with that. Not sure how to pose the question? 
Our users often want to know about:

 an object type [bus]
<img width="828" alt="Screen Shot 2022-09-19 at 8 58 31 AM" src="https://user-images.githubusercontent.com/17605945/191072404-1b4b7092-f549-4bf6-b0ac-48599bfc240f.png">

and a location [Stampede Trail]
<img width="819" alt="Screen Shot 2022-09-19 at 8 58 55 AM" src="https://user-images.githubusercontent.com/17605945/191072482-d63fb80b-81c8-45d4-934a-96da7ca7e923.png">

**Comment by dustymc** [comment]
This is very easy to do in the search UI. Find whatever you want, use this:

<img width="356" alt="Screenshot 2023-02-23 at 3 37 21 PM" src="https://user-images.githubusercontent.com/5720791/221055782-ae5f246d-c117-4120-b0e6-10a70140e1a1.png">

grab the UR, use it in links.

Note that there are also several issues which request eliminating this possibility by requiring a manual submission (so the links would not do anything obvious) or by separating the search and results "sections" in various ways.
